### ISSUE TITLE
Review nixos/about page...

### ISSUE THREAD

**Issue by samueldr** [NixOS/nixos-homepage#230]
Some of the text hasn't been revised since about 9 years ago:

 * https://nixos.org/nixos/about.html

A commit with some formatting changes clouds the complete blame, first link is the current blame, second is blame since that "checkpoint" commit.

* https://github.com/NixOS/nixos-homepage/blame/f74049df65db247ce156022155a966c272ffb96c/nixos/about.tt
 * https://github.com/NixOS/nixos-homepage/blame/e51f72c50e953cb1b6c51083c169fdb62969366f/nixos/about.tt

While some, if not most, of the older text in the page is good, this part may need to be revised:

> Other window managers and **part of Gnome** are also available

That is, unless gnome isn't still fully packaged. It [may cause confusion as to the state of nixos](https://logs.nix.samueldr.com/nixos-chat/2018-07-05#1530821618-1530820807;)

* * *

At the same time, text from other pages may do with a review, a lot can change during almost 10 years.

**Comment by garbas** [comment]
I would even add that about page should be actually part of the landing page

**Comment by davidak** [comment]
yes, the landing page should explain what nixos is and why it's amazing. more technical details can be moved to the manual
